app/models/shelter.rb

```diff
@@ -5,4 +5,14 @@ class Shelter < ApplicationRecord
   def pet_count
     self.pets.count
   end
+
+  def self.sort_alphabetical
+    Shelter.order("name")
+  end
+
+  def self.sort_by_pets
```

Comment:
this method has not been integrated into the app interface, it is incomplete right now.
trying to figure out how to count adoptable pets in active record, not sure what the correct syntax is